bin/ch/ch.cpp

```diff
@@ -450,12 +1014,50 @@ unsigned int WINAPI StaticThreadProc(void *lpParam)
 
 int _cdecl wmain(int argc, __in_ecount(argc) LPWSTR argv[])
 {
-    if (argc < 2)
+    if(argc < 2)
     {
         PrintUsage();
         return EXIT_FAILURE;
     }
 
+    int cpos = 0;
+    for(int i = 0; i < argc; ++i)
+    {
+        if(wcsstr(argv[i], _u("-TTRecord:")) == argv[i])
+        {
+            doTTRecord = true;
```

Comment:
Only one of doTTRecord and doTTDebug can be true at a time right? print error early if both are set?